server.js

```diff
@@ -5,8 +5,7 @@ const path = require('path');
 
 const app = express();
 const port = 1234;
-
-// console.log(process.env.GITHUB_TOKEN);
+axios.defaults.headers.common['Authorization'] = process.env.GITHUB_TOKEN;
```

Comment:
This `axios.defaults.headers.common` will apply the same authorization to all requests, correct? The Amazon s3 API I am working on will be a different auth/credentials. We should see how to make this apply to a list of routes or define individually.

Amazon REST API docs: https://docs.aws.amazon.com/AmazonS3/latest/API/sig-v4-authenticating-requests.html

Reply:
Good call. I'll make a ticket for it. An easy solutions is just to create header objects we can just plop into the `options` space in the axios request.

```js
const atelierHeader = { 
  header: {
    'Authentication': process.env.GITHUB_TOKEN
  }
};

const s3Header = { 
  header: {
    'Authentication': process.env.S3_TOKEN
  }
};
```

Reply:
That sounds good. re-request and I'll approve.